javascript-carousel/main.js

```diff
@@ -0,0 +1,69 @@
+var $next = document.querySelector('.next');
+var $previous = document.querySelector('.previous');
+var $currentImage = document.querySelector('.first');
+var $dotContainer = document.querySelector('.dots');
+
+var clickCounter = 0;
+var clickBack = 0;
+
+$next.addEventListener('click', function (event) {
+  clickCounter++;
+  if (clickCounter === 1) {
+    $currentImage.setAttribute('src', './images/004.png');
+  } else if (clickCounter === 2) {
+    $currentImage.setAttribute('src', './images/007.png');
+  } else if (clickCounter === 3) {
+    $currentImage.setAttribute('src', './images/025.png');
+  } else if (clickCounter === 4) {
+    $currentImage.setAttribute('src', './images/039.png');
+  } else {
+    $currentImage.setAttribute('src', './images/001.png');
+  }
+  changeImage();
+});
+
+$previous.addEventListener('click', function (event) {
+  clickBack++;
+  if (clickBack === 1) {
+    $currentImage.setAttribute('src', './images/039.png');
+  } else if (clickBack === 2) {
+    $currentImage.setAttribute('src', './images/025.png');
+  } else if (clickBack === 3) {
+    $currentImage.setAttribute('src', './images/007.png');
+  } else if (clickBack === 4) {
+    $currentImage.setAttribute('src', './images/004.png');
+  } else {
+    $currentImage.setAttribute('src', './images/001.png');
+  }
+});
```

Comment:
Instead of using a `clickCounter` or `clickBack`, use your `i` and `imageSlider`. Then you can do
```js
$currentImage.setAttribute('src', imageSlider[i]);
```